src/components/Autosuggest/index.tsx

```diff
@@ -162,6 +194,7 @@ const Autosuggest = ({
       <SuggestionsList
         activeIndex={state.activeIndex}
         filteredSuggestions={state.filteredSuggestions}
+        deleteForwardUrls={deleteForwardUrls}
         onMouseDown={onMouseDown}
       />
     </div>
```

Comment:
There's a strange bug. Any time I do something with a forward URL, the selected endpoint in the AutoForward dropdown resets to the first item in the list. This happens when I:
* Forward a webhook
* Delete a forward URL from the main webhook panel
* Delete a forward URL from the auto-forward panel

Here's a recording that demonstrates this: https://recordit.co/9QNBWX9GDv

Reply:
The issue is resolved. The selection doesn't get changed when we do the actions mentioned above anymore.